keywords/exceptions.py

```diff
@@ -64,3 +64,7 @@ class DockerError(Error):
 
 class FeatureSupportedError(Error):
     pass
+
+
+class ConnectionError(Error):
```

Comment:
what is the class for?

Reply:
In test_doc_expiry_exceeded_200days, I need to catch the connection Error in python test. I need to add this ConnectionError to let Python script to identify as exception